### PR TITLE
added credential table for user federation

### PR DESCRIPTION
**Pull request by edewit**
same as in the old console when credentials are federated by ldap

### CODE REVIEW

File: apps/admin-ui/src/user/UserCredentials.tsx [hunk 1]
```diff
@@ -490,26 +491,31 @@ export const UserCredentials = ({ user }: UserCredentialsProps) => {
...
-          }
-        />
       )}
+      {(user.federationLink || user.origin) && (
+        <FederatedCredentials user={user} onSetPassword={toggleModal} />
+      )}
+      {groupedUserCredentials.length === 0 &&
+        !(user.federationLink || user.origin) && (
+          <ListEmptyState
+            hasIcon={true}
```
Comment (jonkoops) [inline apps/admin-ui/src/user/UserCredentials.tsx:501]:
```suggestion
            hasIcon
```

File: apps/admin-ui/src/user/user-credentials/FederatedCredentials.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                <b>{credential}</b>
+              </Td>
+              <Td>
+                <Button
+                  variant="link"
+                  component={(props) => (
+                    <Link
+                      {...props}
+                      to={
+                        access.hasAccess("view-realm")
```
Comment (jonkoops) [inline apps/admin-ui/src/user/user-credentials/FederatedCredentials.tsx:89]: Should this link not be disabled instead when there is no access?

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: apps/admin-ui/src/user/user-credentials/FederatedCredentials.tsx (hunk 2 above)
Comment (edewit) [reply apps/admin-ui/src/user/user-credentials/FederatedCredentials.tsx:89]: yeah guess that's better this is what the old console did... but I'll change it

**Comment by jonkoops** [comment]
@edewit the tests seem to keep failing, can you take a look?

**Pull request closed without merging**